src/utils/queries.js

```diff
@@ -0,0 +1,15 @@
+import { Q } from 'cozy-client'
+
+import { FILES_DOCTYPE } from '../doctypes'
+
+export const queryAllPapers = {
+  definition: () =>
+    Q(FILES_DOCTYPE).where({
+      type: 'file',
+      trashed: false,
+      [`metadata.qualificationsAttributes`]: true
```

Comment:
C'est quoi cet attribut ? Il ne semble pas dans la doc https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files_metadata.md . Il est setté quand et comment ? 

Si l'idée est de récupérer tous les papiers concernant l'application, alors la requête devrait plutôt se baser sur la définition en JSON des papiers pour réquéter. Typiquement si on prend la définition que tu as aujourd'hui, tu boucles sur le JSON, tu récupères tous les labels des qualif et tu fais un where({metadata.qualification.label : { $in: labels }}

Reply:
1/ Je pensais ajouter une nouvelle metadata à la création d'un Papier (*qualificationsAttributes*), qui sera définie par le [fichier de définitions des Papiers (L14)](https://github.com/cozy/mespapiers/pull/5/files#diff-653aed11a0492314de4e6340a2bae9d824de79dc9fbef0fd66996bfe16e29b98R14)

2/ Pour moi un fichier qualifié n'est pas nécessairement un Papier, par contre un Papier est forcément qualifié.
C'est pour ça que je récupère les Papiers en filtrant sur la présence du metadata *qualificationsAttributes*